cpu/stm32/Kconfig

```diff
@@ -565,4 +121,17 @@ config CPU_MODEL
 config CPU
     default "stm32" if CPU_STM32
 
+source "$(RIOTCPU)/stm32/Kconfig.f0"
```

Comment:
@leandrolanzieri, do you think it's ok this way or should I conditionally source the family Kconfig file (is it possible ?)

Reply:
Adding an `if` around a `source` directive in Kconfig does not do a conditional inclusion, but applies the condition as a dependency to all the symbols defined inside. In this case it would mean to make `CPU_FAM_F0` (which is defined inside `Kconfig.f0`) to depend on itself.

The files are split just for a matter of organization, it does not hurt to include all of them to the tree always. I think what you did is OK, we don't need the conditionals here.

Reply:
> In this case it would mean to make CPU_FAM_F0 (which is defined inside Kconfig.f0) to depend on itself.

Yeah, but if that would have been possible, I would have moved the definitions of the families back in this file.

> I think what you did is OK, we don't need the conditionals here.

Ok, great!